.github/workflows/generate_site_and_deploy.yaml

```diff
@@ -3,10 +3,6 @@ name: generate_site_and_deploy
 on:
   workflow_dispatch:
   push:
-    paths:
-      - src/**
```

Comment:
Recommend to ignore /too and README.md and some several files.

But just removing is also OK since #113 👍